zingolib/src/wallet/transaction_record.rs

```diff
@@ -324,9 +324,39 @@ impl TransactionRecord {
     }
 }
 
+#[cfg(feature = "test-features")]
```

Comment:
I think cfg test will silence the dead code warnings, assuming that this code is called in tests. test-features is for when we want to call the code from another crate's tests, and this code is crate-local.

```suggestion
#[cfg(test)]
```